src/index.html

```diff
@@ -9,10 +9,51 @@
     <title>Stars</title>
     <link
       rel="stylesheet"
-      href="./style.css"
+      href="./styles/style.css"
     />
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+    <section class="stars stars--0">
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+      <span class="stars__star"></span>
+    </section>
+    <section class="stars stars--1">
```

Comment:
Add the blank line between the blocks
```suggestion
    </section>
    
    <section class="stars stars--1">
```